.travis.yml

```diff
@@ -1,18 +1,23 @@
-dist: xenial
```

Comment:
Why did you removed it?

Reply:
We don't care the version now as we are using docker now.

Reply:
Since we are using docker now, we don't care about which ubuntu version travis is using.